controllers/helpers.go

```diff
@@ -32,6 +32,78 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+// filterContainerIDs filters out any containers that may not be intended for disruption in a multi-container disruption
+// this function returns 3 string slices:
+// containers: The updated list of containers with the filtered out containers removed
+// removed: The list of containers that were removed from the list containers, used later for logging purposes
+// reasoning: A list of strings used later for logging the reason as to why the corresponding container in removed was removed
```

Comment:
purely optional, but since `removed` and `reasoning` are so inextricably tied together, you could return a slice of a new struct that represents a removed/reasoning tuple, and reduce the return values to just two slices

Reply:
It looks like `reasoning` is always the same string, right? In that case, why not remove this return value and just use that string directly at the point of logging?

Reply:
Reasoning may be different in the future, for now there is only one value for it, but in the future if we filter out containers for other reasons, that reasoning will most likely change, thats why I've set that string to its own set. I do like the idea of putting these in a struct and returning a slice.

Reply:
This comment needs updated now, but 👍